test/AzureExtension/AzureExtension.Test.csproj

```diff
@@ -5,8 +5,8 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <RootNamespace>AzureExtension.Test</RootNamespace>
-    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
-    <PublishProfile>Properties\PublishProfiles\win10-$(Platform).pubxml</PublishProfile>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>

```

Comment:
You may want to double check that the tests run without issue.

I see we have an `AzureExtension` reference which has a DevHome SDK reference.  We previously saw in https://github.com/microsoft/devhome/pull/2156 that if any of the dependent packages were still using the old runtime identifier format, they may fail to get packaged up once we move to the new format.  The previous DevHome SDK NuGet until the .NET 8 change was using the old runtime identifier format.  You probably want to make sure you are updated to the new DevHome SDK NuGet that uses the new 'win-x64' format.  It is possible that the test still passes without this if it doesn't actually use the SDK.

Reply:
I see what you mean.  I followed the changes in the PR you linked.  Moved the identifiers to win10-x.  Solution built just fine.